sys/include/net/lorawan/hdr.h

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_lorawan_hdr    LoRaWAN header
+ * @ingroup     net_lorawan
+ * @brief       LoRaWAN header types and helper functions
+ * @{
+ *
+ * @file
+ * @brief   LoRaWAN header type and helper function definitions
+ *
+ * @author  José Ignacio Alamos <jose.alamos@inria.cl>
+ */
+
+#ifndef LORAWAN_HDR_H
+#define LORAWAN_HDR_H
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "byteorder.h"
+#include "net/loramac.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Data type to represent a LoRaWAN packet header
+ *
+ * @details This definition includes MHDR and FHDR in the same structure.
```

Comment:
I don't think that `@details` is needed (here and below). At least it is hardly used in the RIOT docu else where

Reply:
removed